Cargo.toml

```diff
@@ -241,5 +241,6 @@ codegen-units = 1
 #
 # Or for another Git dependency:
 #
-# [patch."https://github.com/servo/<repository>"]
-# <crate> = { path = "/path/to/local/checkout" }
+# [patch."https://github.com/servo/rust-content-security-policy"]
```

Comment:
Could you revert these changes?

Reply:
Do you mind I keep them here so in follow-up PRs I can easily turn them back on? Juggling across dependencies for local, CI and when merging is quite a lot. Since they are commented out, I don't think this is a big issue, is it?

Reply:
If you re-add the model comments that were changed then I'm fine with it.